src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -361,8 +361,9 @@ private void handleCreateConversation(int chatId) {
     int listflags = 0;
     if (archive) {
       listflags |= DcContext.DC_GCL_ARCHIVED_ONLY;
-    } else if (forwarding) {
-      listflags |= DcContext.DC_GCL_FOR_FORWARDING;
+    } else if (RelayUtil.isRelayingMessageContent(getActivity())) {
+      listflags |= DcContext.DC_GCL_NO_SPECIALS;
```

Comment:
why not just use `DcContext.DC_GCL_FOR_FORWARDING` for both, forwarding and sharing?

Reply:
> also, the `forwarding` argument can be probably removed when using the RelayUtil, or?

What do you mean with "The forwarding argument"? The one I already removed?

Reply:
> why not just use DcContext.DC_GCL_FOR_FORWARDING for both, forwarding and sharing?

Yes, makes sense, I changed this.

> also, the `forwarding` argument can be probably removed when using the RelayUtil, or?

What do you mean with "The forwarding argument"? The one I already removed?